### PR TITLE
Init command for basic_flow example

### PR DESCRIPTION
**Pull request by toszo**
Previously "make init" was missing for this example

### CODE REVIEW

File: examples/basic_flow/Makefile [hunk 1]
```diff
@@ -15,6 +16,8 @@ export
 
 all: apply
 
+init: init-azbi init-azks
```
Comment (sk4zuzu) [inline examples/basic_flow/Makefile:19]: This seems to be incorrect. I think (in principle) `init-azks` needs to be executed after `apply-azbi` is done and successful. :thinking:

File: examples/basic_flow/Makefile [hunk 2]
```diff
@@ -1,9 +1,10 @@
 AZBI := epiphanyplatform/azbi:0.0.1
 AZKS := epiphanyplatform/azks:0.0.1
 
+makefile_dir_path := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```
Comment (sk4zuzu) [inline examples/basic_flow/Makefile:4]: This is the "optimized" way (uses built-in functions). :+1:
```suggestion
makefile_dir_path := $(patsubst %/,%,$(dir $(realpath $(firstword $(MAKEFILE_LIST)))))
```

**Review by sk4zuzu — CHANGES_REQUESTED** [review]

File: examples/basic_flow/Makefile (hunk 1 above)
Comment (mkyc) [reply examples/basic_flow/Makefile:19]: That is definitely incorrect @toszo. Please have a look [here](https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/howto/modules.md#user-point-of-view) where it's described that you need to perform `init`, `plan` and `apply` cycles for each module.

File: examples/basic_flow/Makefile [hunk 3]
```diff
@@ -1,9 +1,10 @@
 AZBI := epiphanyplatform/azbi:0.0.1
 AZKS := epiphanyplatform/azks:0.0.1
 
+makefile_dir_path := $(patsubst %/,%,$(dir $(realpath $(firstword $(MAKEFILE_LIST)))))
 #makes it easier to replace the value
 M_NAME            ?= mop-module-tests
-M_SHARED_EXTERNAL ?= /tmp/$(M_NAME)/shared
+M_SHARED_EXTERNAL ?= $(makefile_dir_path)/$(M_NAME)/shared
```
Comment (mkyc) [inline examples/basic_flow/Makefile:7]: @sk4zuzu approach with `/tmp/azks/shared` was aimed (AFAIU) to have some storage place for runtime. I think changing it to something else than `/tmp/` might make some sense but I would not make it `$(makefile_dir_path)/$(M_NAME)/shared` but rather `$(makefile_dir_path)/shared`. It's not shared directory of this module, but for all the elements installed by this example. Also I would add this directory to `.gitignore`.

**Review by mkyc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**